csun-metalab/nomi
# NAM-355 -- UI: Implement mockup design for desktop login

# Description
Created the desktop login page based off the mock-up.
  
# Testing
1. Test to see if all the proper functionality works in desktop and mobile view.
2. Check to see if there is any content that breaks the boundary or not in the right position.
3. Ensure the login page is visually appealing when scaling the browser.
4. Disable your images to see if all the proper accessibility tags are on the images.
5. Ensure tab functionality and other accessibility features work.

# Installation
You may need to download the web developer chrome extension or other variants to disable images on a web page.
